frontend/src/libs/helpers/get-formatted-time.helper.ts

```diff
@@ -0,0 +1,13 @@
+import { format } from 'date-fns';
+
+const EPOCH_MILLISECONDS = 0;
+const TIME_FORMAT = 'mm:ss';
```

Comment:
we should pass format where we call this function, create a type for the format, and then just pass string where it's needed
```suggestion
type TimeFormat = 'mm:ss';
```